Plugins/Json/include/Acts/Plugins/Json/AlgebraJsonConverter.hpp

```diff
@@ -28,7 +28,7 @@ struct Options {
   /// Write the identity transform explicitly
   bool writeIdentity = false;
   /// Apply a transpose to flip column/row order
-  bool transpose = true;
+  bool transpose = false;
```

Comment:
Ha, interesting, there is a difference between `detray` style writing - and core writing.